ci/workflows/pull-request.yml

```diff
@@ -227,7 +197,18 @@ steps:
   - label: "check"
     command: "ci/scripts/check.sh"
     plugins:
-      - gencer/cache#v2.4.10: *cargo-cache
+      - gencer/cache#v2.4.10:
```

Comment:
Any reason for still keeping this?

Reply:
59s vs 4s 🤔 
<img width="841" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/d6944636-0736-4c3c-bf49-6ee5518e77e2">